Library/Homebrew/system_command.rb

```diff
@@ -276,9 +273,56 @@ def each_output_line(&block)
   rescue Interrupt
     Process.kill("INT", raw_wait_thr.pid) if raw_wait_thr && !sudo?
     raise Interrupt
-  rescue SystemCallError => e
-    @status = $CHILD_STATUS
-    @output << [:stderr, e.message]
+  ensure
+    raw_stdin&.close
+    raw_stdout&.close
+    raw_stderr&.close
+  end
+
+  sig {
+    params(
+      env:        T::Hash[String, String],
+      executable: String,
+      args:       String,
+      options:    T.untyped,
```

Comment:
I _think_ we can restrict this to be a `Hash` at least.

Reply:
Kwargs are treated differently in Sorbet:
https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Asig%20%7Bparams%28kwargs%3A%20Integer%29.void%7D%0Adef%20foo%28**kwargs%29%0A%20%20T.reveal_type%28kwargs%29%0Aend%0A